frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/activity-card/activity-card.tsx

```diff
@@ -21,12 +21,35 @@ import {
   DashboardItemProps,
   withDashboardResources,
 } from '@console/internal/components/dashboard/with-dashboard-resources';
-import { CEPH_STORAGE_NAMESPACE } from '../../../../constants/index';
+import { referenceForModel } from '@console/internal/module/k8s/k8s';
+import { SubscriptionModel } from '@console/operator-lifecycle-manager';
+import { CEPH_STORAGE_NAMESPACE, OCS_OPERATOR } from '../../../../constants/index';
 import { DATA_RESILIENCY_QUERY, StorageDashboardQuery } from '../../../../constants/queries';
 import { getResiliencyProgress } from '../../../../utils';
+import { OCSServiceModel } from '../../../../models';
+import { isClusterExpandActivity } from './cluster-expand-activity';
+import { isOCSUpgradeActivity } from './ocs-upgrade-activity';
 import './activity-card.scss';
 
 const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+const subscriptionResource: FirehoseResource = {
+  isList: true,
+  kind: referenceForModel(SubscriptionModel),
+  namespaced: false,
+  prop: 'subs',
+};
+
+const storageClusterResource: FirehoseResource = {
+  isList: true,
+  kind: referenceForModel(OCSServiceModel),
+  namespaced: false,
+  prop: 'storage-cluster',
+};
+
+export const getOCSSubscription = (subscriptions: FirehoseResult): K8sResourceKind => {
+  const itemsData: K8sResourceKind[] = _.get(subscriptions, 'data');
```

Comment:
you can use TS 3.7 optional chaining - applies for all usages of `_.get`

Reply:
I was waiting for this, now I ma looking for all instances , to nail down with the hammer :D

Reply:
![Screenshot from 2020-01-21 18-29-05](https://user-images.githubusercontent.com/25664409/72814598-51e05b00-3c8b-11ea-925d-82efc0ae943d.png)